database/factories/ModelFactory.php

```diff
@@ -16,6 +18,25 @@
 |
 */
 
+// Event Factory
+$factory->define(Event::class, function (Generator $faker) {
+    return [
+        'northstar_id' => str_random(24),
+        'submission_type' => 'signup',
```

Comment:
Makes sense! In the future it could make sense to have different [factory states](https://laravel.com/docs/5.4/database-testing#writing-factories) for each "type" of event. For example, we do this in Northstar so we can easily make User models w/ different roles.